### PR TITLE
Feature/eventbridge v2 use arn util for arns

### PR DESCRIPTION
**Pull request by maxhoheiser**
## Motivation
Using the arn util to define arns via and access them via a property of the dataclasses defined for each top-level entity of events (bus, rule, archive, replay) is preferable to having the logic of creating arns located in the model files of event bridge.

### CODE REVIEW

File: localstack-core/localstack/services/events/models.py [hunk 1]
```diff
@@ -38,7 +38,12 @@
     CrossRegionAttribute,
     LocalAttribute,
 )
-from localstack.utils.aws.arns import events_archive_arn, events_replay_arn
+from localstack.utils.aws.arns import (
+    event_bus_arn,
```
Comment (joe4dev) [inline localstack-core/localstack/services/events/models.py:42]: nit: intentionally `event_` vs `events_` prefix

File: localstack-core/localstack/services/events/models.py [hunk 2]
```diff
@@ -136,6 +136,10 @@ def __post_init__(self):
         if self.state is None:
             self.state = RuleState.ENABLED
 
+    @property
+    def arn(self) -> Arn:
```
Comment (joe4dev) [inline localstack-core/localstack/services/events/models.py:140]: nice that we don't need to save the redundant field 👍

**Review by joe4dev — APPROVED** [review]
neat refactoring 👍

File: localstack-core/localstack/services/events/models.py (hunk 1 above)
Comment (maxhoheiser) [reply localstack-core/localstack/services/events/models.py:42]: Yes, internally everything is called event_ .... but externally anything related to event bridge uses the shorthand events_ ... here particular only related to defining the respective ARNs for event_bridge

**Pull request merged**